neurodamus/ngv.py

```diff
@@ -191,22 +234,27 @@ def _show_mcd(sec):
     #        )
 
     def set_pointers(self):
-        glut_list = self._glut_list
         c = self._cellref
-        index = 0
+        # the endfeet are not included in all as they are added later.
+        # I still do not know exactly when the pointers need to be
```

Comment:
is this a note that should be acted on before this is merged?

Reply:
no. I left if for posterity in case I figure out why these pointers need to be reassigned. Maybe creating an issue is better

Reply:
#263

Reply:
I think it needs to be figured out before this is merged because I'm not sure how this is comment actionable

Reply:
I do not know how to do something about it at the moment. The reason is probably buried in stdinit. I opened an issue to ask

Reply:
Tough one, I guess we go w/ this for now.